docs/_documentations/mdt-vsc-nodejsprofiling.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Installing and running the Codewind language server for Node.js profiling
+description: Installing and running the Codewind language server for Node.js profiling
+keywords: Node.js, Codewind, server, language, languages, code, code highlighting, highlighting, profiling, Visual Studio Code, VS Code, JavaScript
+duration: 1 minute
+permalink: mdt-vsc-nodejsprofiling
+type: document
+---
+
+# Installing and running the Codewind language server for Node.js profiling
+The Codewind language server for Node.js profiling annotates your Node.js code with code highlighting. Code highlighting uses the profiling data gathered through Codewind load testing to highlight and show the relative time that is spent in JavaScript functions.
+
+## Running the extension with Visual Studio Code (VS Code)
+1. Open a local project that you created with [Codewind](mdt-vsc-getting-started.html) and profiled by using the [performance test](guide_performance.html) feature. Opening the project creates profiling data in a `load-test/[datestamp]/profiling.json` file in your Codewind project.
+2. In VS Code, open a JavaScript file in your project. The extension highlights any lines that it finds in the profiling data and annotates them to show how often they were seen and where they were called from.
+
+## Running the extension with VS Code
```

Comment:
@tobespc I just noticed...lines 14 and 18 have the same heading. Should we combine these two sections, or how should we title the headings so that they are distinct? Thank you!